lib/context.js

```diff
@@ -370,31 +379,42 @@ api.createTermDefinition = (activeCtx, localCtx, term, defined) => {
   }
 
   if('@container' in value) {
-    const container = value['@container'];
-    if(container !== '@list' && container !== '@set' &&
-      container !== '@index' && container !== '@language') {
-      let isValid = false;
-      const validContainers = ['@list', '@set', '@index', '@language'];
-      // JSON-LD 1.1 support
-      if(activeCtx['@version'] === 1.1) {
-        // || processingMode === 'jsonld-1.1') {
-        validContainers.push('@graph');
-        if(container === '@graph' ||
-          (_isArray(container) && container.length === 2 &&
-          container.includes('@graph') && container.includes('@set'))) {
-          isValid = true;
-        }
-      }
-      if(!isValid) {
-        throw new JsonLdError(
-          'Invalid JSON-LD syntax; @context @container value must be ' +
-          'one of the following: ' + validContainers.join(', '),
-          'jsonld.SyntaxError',
-          {code: 'invalid container mapping', context: localCtx});
-      }
+    // Normalize container to an array form
+    const container = _isString(value['@container']) ? [value['@container']] : (value['@container'] || []);
+    const validContainers = ['@list', '@set', '@index', '@language'];
```

Comment:
Should probably use a `Set` for `validContainers` instead of an array -- and then use `.has` instead of `.includes`. But just a minor nit. Just pointing out that we should make better use of the newer ES6/ES7/whatever features.

Reply:
Looks like node 6 BUNDLER=webpack build doesn't like the use of `Set`: https://travis-ci.org/gkellogg/jsonld.js/jobs/326948773. Is there some transformation that is supposed to happen automatically?

Reply:
babel handles syntax, and apis come from core-js.  In theory, you would just add `core-js/fn/set` to `webpack.config.js` and `tests/test-karma.js` along with the other core-js things.  But I just tried and karma tests still fail even though looks like Set code is in the built code.

Reply:
I'm going to back out the change to use `Set`, which could probably be used in other places anywhere, and it's really the point of this branch.

Reply:
Ok... the fix I suggested mostly works.  It's just a few error tests that fail for some reason and only in PhantomJS.